R/module_bookmark_manager.R

```diff
@@ -38,51 +38,42 @@
 #' @name module_bookmark_manager
 #' @keywords internal
 #'
-bookmark_manager_ui <- function(id) {
+ui_bookmark_panel <- function(id, modules) {
   ns <- NS(id)
-  uiOutput(ns("bookmark_button"), inline = TRUE)
+
+  bookmark_option <- get_bookmarking_option()
+  is_unbookmarkable <- need_bookmarking(modules)
+
+  # Render bookmark warnings count
+  if (!all(is_unbookmarkable) && identical(bookmark_option, "server")) {
```

Comment:
else throw a warning (or message) that some modules are not bookmarkable
enumerate which ones